kernel/Dockerfile

```diff
@@ -62,8 +63,18 @@ RUN curl -fsSLO ${KERNEL_SHA256_SUMS} && \
     gpg2 --verify linux-${KERNEL_VERSION}.tar.sign linux-${KERNEL_VERSION}.tar && \
     cat linux-${KERNEL_VERSION}.tar | tar --absolute-names -x && mv /linux-${KERNEL_VERSION} /linux
 
-# Apply local patches if present
 WORKDIR /linux
+# Apply local specific patches if present
+RUN set -e && \
+    if [ -n "${EXTRA}" ] && [ -d /patches-${KERNEL_SERIES}${EXTRA} ]; then \
+	echo "Patching ${EXTRA} kernel"; \
+	for patch in /patches-${KERNEL_SERIES}${EXTRA}/*.patch; do \
```

Comment:
So, the series file is no longer needed? I recall an earlier version used that. It's great if we don't need it!

Reply:
@rn This time I manually renamed these -rt patch files prefixed by the numer term orderly, so now this can work without that orginal series file. I think this can make our life easy.

Reply:
OK, thanks. Can this be automated? Or even better is there a kernel git branch somewhere with the RT patches? That way We can just to a `git format-patch` on an update. https://git.kernel.org/pub/scm/linux/kernel/git/rt/linux-stable-rt.git doesn't seem to be structure that way...

If there isn't a suitable git tree, it might be good to add a script under `./scripts` to convert the patches as there are in the tarball into how we'd like them in the patches directory

Reply:
@rn Yes, I use one simple script to rename those -rt patches. Maybe I can update this later with another pull request?